src/helpers/reports.js

```diff
@@ -43,10 +43,10 @@ export const supportLevelColors = {
   below: '#c93d82',
 };
 
-export const getSupportLevel = (percentile) => {
+export const getSupportLevel = (grade, percentile, rawScore, taskId) => {
   let support_level = null;
   let tag_color = null;
-  if (percentile !== undefined) {
+  if (percentile !== undefined && grade < 6 && !['swr, pa'].includes(taskId)) {
```

Comment:
I don't understand why we check the `taskId` here as well. This would suggest that we don't want to give support categories to elementary aged students who have a percentile score for SWR or PA.

Whatever the answer to the above question is, shouldn't this line also be
```suggestion
  if (percentile !== undefined && grade < 6 && !['swr', 'pa'].includes(taskId)) {
```

Reply:
I think the explicit check on task IDs can be removed. My reasoning is that this is the same situation as elementary school students who take letter. We don't have norms for letter so we don't assign them a category. When they enter this function, they won't have `percentile` defined, so they will not get a support category. And I believe the logic is the same for middle and high schoolers. Does that make sense?